.travis.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+function build_container(){
+    docker build -t travis/image-inspector-base .
+    cat > Dockerfile.travis <<EOF
+FROM travis/image-inspector-base
+RUN yum install -y \
+    git \
+    which \
+    make
+RUN yum remove -y golang
+RUN curl -O https://storage.googleapis.com/golang/go1.8.3.linux-amd64.tar.gz && \
+    tar -C /usr/local -xzf go1.8.3.linux-amd64.tar.gz && \
+    rm -f  go1.8.3.linux-amd64.tar.gz
```

Comment:
I'll need to change one of the tests because `reflect.DeepEquals` was changed in go 1.8 (making it much easier to compare objects that share a type). I was waiting for the merge of #59 but I can go ahead and make that change now